pkg/virt-api/webhooks/validating-webhook/admitters/migration-update-admitter_test.go

```diff
@@ -169,7 +224,8 @@ var _ = Describe("Validating MigrationUpdate Admitter", func() {
 
 		admitter := &admitters.MigrationUpdateAdmitter{}
 		resp := admitter.Admit(context.Background(), ar)
-		Expect(resp.Allowed).To(BeTrue())
+
+		Expect(resp).To(Equal(allowedAdmissionResponse()))
```

Comment:
I find it a bit odd that we have `allowedAdmissionResponse` to begin with, consider using `validating_webhooks` `NewPassingAdmissionResponse` as it achieves the exact same.

Reply:
`allowedAdmissionResponse()` resides in the admitters_test package, and is independent of `validating_webhooks NewPassingAdmissionResponse`.

If we've used the same function twice, the test wasn't independent and would always pass.

Reply:
Ack.